scripts/render-notes.py

```diff
@@ -86,30 +102,39 @@ def render_notes(
         },
     }
 
+    template_base = os.path.join(os.path.dirname(sys.argv[0]), "..")
```

Comment:
This might be easier to read with:

```
     render_files = {
         "changelog_master": {
-            "template": "./scripts/templates/changelog_master.xml",
+            "template": "changelog_master.xml",
             "outfile": f"./app-{application}/src/main/res/raw/changelog_master.xml",
             "render_data": render_data["releases"][version],
         },
         "changelog": {
-            "template": "./scripts/templates/changelog.txt",
+            "template": "changelog.txt",
             "outfile": f"./app-metadata/{applicationid}/en-US/changelogs/{versioncode}.txt",
             "render_data": render_data["releases"][version],
         },
         "changelog_long": {
-            "template": "./scripts/templates/changelog_long.txt",
+            "template": "changelog_long.txt",
             "outfile": longform_file,
             "render_data": render_data["releases"][version],
         },
     }
+    template_base = os.path.join(os.path.dirname(sys.argv[0]), "templates")
 
     for render_file in render_files:
-        with open(render_files[render_file]["template"], "r") as file:
+        with open(os.path.join(template_base, render_files[render_file]["template"]), "r") as file:
```